### ISSUE TITLE
Get rid of prepareStatement overhead

### ISSUE THREAD

**Issue by eivanov89** [ydb-platform/tpcc#18]
In a multistep transaction we send prepare transaction requests multiple times (obtaining a session each time, which can be blocking).

Alex, you told you know, how to avoid this stuff. Please, have a look again.

**Comment by alex268** [comment]
We already added a support of caching of prepared queries to [JDBC Driver](https://github.com/ydb-platform/ydb-jdbc-driver/pull/37) so there is no overhead for requests except for first

**Comment by eivanov89** [comment]
> We already added a support of caching of prepared queries to https://github.com/ydb-platform/ydb-jdbc-driver/pull/37 so there is no overhead for requests except for first

So, do we need just to update JDBC driver (as planned)?

**Comment by alex268** [comment]
Yes, this issue will be fixed after updating to new version of JDBC driver
